pkg/matcher/README.md

```diff
@@ -4,12 +4,25 @@
   * glob
   * regexp
   * simple patterns
-  
+
+Depending on the symbol at the start of the string, the `matcher` will use one of the supported formats. 
+
+matcher | short format | long format
+ ------ | ------------ | ----------
+ string | ` =` | `string`
+ glob | `*` | `glob`
+ regexp | `~` | `regexp`
+ simple patterns | | `simple patterns`
```

Comment:
`simple patterns` => `simple_patterns`

Reply:
i would add negative matchers here too

Reply:
and a description, like i have it there

> https://github.com/netdata/go.d.plugin/tree/master/pkg/prometheus/selector#syntax-1

but table is better then list in this case, yes